content-src/asrouter/asrouter-content.jsx

```diff
@@ -299,14 +312,22 @@ export class ASRouterUISurface extends React.PureComponent {
   render() {
     const {message, bundle} = this.state;
     if (!message.id && !bundle.template) { return null; }
-    return (
-      <React.Fragment>
-        {this.renderPreviewBanner()}
-        {this.renderFirstRunOverlay()}
-        {this.renderOnboarding()}
-        {this.renderSnippets()}
-      </React.Fragment>
-    );
+
+    if (!TEMPLATES_BELOW_SEARCH.includes(message.template)) {
+      return ReactDOM.createPortal(
+        <React.Fragment>
+          {this.renderPreviewBanner()}
+          {this.renderFirstRunOverlay()}
+          {this.renderOnboarding()}
+          {this.renderSnippets()}
+        </React.Fragment>,
+        this.footerPortal
```

Comment:
What's the  behaviour of this if `footerPortal` is `undefined`? (like in tests for whatever

Reply:
I had to put in a fake portal container for tests.

Reply:
ok cool